EngineInterface/images/imapConnect.py

```diff
@@ -0,0 +1,115 @@
+import email
+import imaplib
+import os
+import smtplib
+import mimetypes
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.mime.base import MIMEBase
+
+def sendEmail():

```

Comment:
Might make this take the address it sends to as part of the function
```suggestion
def sendEmail(toAddr):
```

Reply:
Removed unnecessary files alls that is left now is updating comments in order for QA to write tests as per last review